code/cu_manager/src/wattrex_battery_cycler_cu_manager/cu_broker_client.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python3
+"""
+Wrapper for the MQTT client
+"""
+#######################        MANDATORY IMPORTS         #######################
+
+#######################         GENERIC IMPORTS          #######################
+from typing import Callable, List
+from pickle import dumps, loads
+from uuid import getnode
+
+#######################       THIRD PARTY IMPORTS        #######################
+
+#######################    SYSTEM ABSTRACTION IMPORTS    #######################
+from system_logger_tool import sys_log_logger_get_module_logger, SysLogLoggerC, Logger
+
+#######################       LOGGER CONFIGURATION       #######################
+if __name__ == '__main__':
+    cycler_logger = SysLogLoggerC(file_log_levels='../log_config.yaml')
+log: Logger = sys_log_logger_get_module_logger(__name__)
+
+#######################          PROJECT IMPORTS         #######################
+from wattrex_battery_cycler_datatypes.comm_data  import CommDataCuC,\
+    CommDataDeviceC,CommDataHeartbeatC, CommDataRegisterTypeE
+
+from wattrex_driver_mqtt import DrvMqttDriverC
+
+#######################          MODULE IMPORTS          #######################
+
+#######################              ENUMS               #######################
+_REGISTER_TOPIC = '/register'
+_INFORM_TOPIC = '/inform_reg'
+_SUFFIX_TX_DET_DEV = '/detected_dev'
+_SUFFIX_TX_HB = '/heartbeat'
+_SUFFIX_RX_DET = '/req_detect'
+_SUFFIX_RX_LAUNCH = '/launch'
+
+_MAC: int = getnode()
```

Comment:
Store it on cu_manager and pass it to the broker client on initialization

Reply:
Noo, you should pass it to broker init function

Reply:
Done. MAC retrieved in register.py when called from cu_manager and passed to cu_brocker when registering or subscribing